docs/guide/troubleshooting.md

```diff
@@ -12,7 +12,7 @@ The path to your project folder may include `&`, which doesn't work with `npm` o
 
 You will need to either:
 
-- Switch to another package manager (e.g. `pnpm`, `yarn`)
+- Switch to another package manager (e.g. `pnpm`, `yarn`, `bun`)
```

Comment:
IIUC this error only happens on non-WSL Windows environment and bun doesn't work in that environment. So I guess bun cannot simply added here.

Reply:
Removed